sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -29,13 +38,40 @@ export interface LinkEntityOptions {
   audience?: string;
 }
 
+/**
+ * A simple grouping of the sender and receiver options. Only used
+ * with the ManagementClient today.
+ *
+ * @internal
+ * @ignore
+ */
+export interface RequestResponseLinkOptions {
+  senderOptions: SenderOptions;
+  receiverOptions: ReceiverOptions;
+  name?: string;
```

Comment:
Is this name needed? I dont see us passing it anywhere

Reply:
So _no_ - there's some generic logic in the init() method to let the user override the name via options (so it's required to exist) but it's optional because for the management client we're not filling it in.

So it's vestigial but kind of needed for the contract.